pyomo/contrib/community_detection/tests/test_detection.py

```diff
@@ -0,0 +1,321 @@
+"""Community Detection Test File - Rahul Joglekar"""
+
+# Structure for this file was adapted from:
+#  ___________________________________________________________________________
+#
+#  Pyomo: Python Optimization Modeling Objects
+#  Copyright 2017 National Technology and Engineering Solutions of Sandia, LLC
+#  Under the terms of Contract DE-NA0003525 with National Technology and
+#  Engineering Solutions of Sandia, LLC, the U.S. Government retains certain
+#  rights in this software.
+#  This software is distributed under the 3-clause BSD License.
+#  ___________________________________________________________________________
+
+from __future__ import division
+import pyutilib.th as unittest
+
+from pyomo.environ import *
+from pyomo.contrib.community_detection.detection import *
+
+from pyomo.solvers.tests.models.LP_unbounded import LP_unbounded
+from pyomo.solvers.tests.models.QP_simple import QP_simple
+from pyomo.solvers.tests.models.LP_inactive_index import LP_inactive_index
+from pyomo.solvers.tests.models.SOS1_simple import SOS1_simple
+
+
+class TestDecomposition(unittest.TestCase):
+
```

Comment:
define a variable `random_seed=5` here and use it below instead of hardcoding the 5 in every instance